mtproto/ping.go

```diff
@@ -36,10 +44,10 @@ func (c *Conn) handlePong(b *bin.Buffer) error {
 	c.log.Debug("Pong")
 
 	c.pingMux.Lock()
-	f, ok := c.ping[pong.PingID]
+	ch, ok := c.ping[pong.PingID]
 	c.pingMux.Unlock()
 	if ok {
-		f()
+		close(ch)
```

Comment:
Will it panic on double pong for same PingID?

Reply:
Yes :0